### ISSUE TITLE
Physical Case Issue

### ISSUE THREAD

**Issue by IndigoHaze** [CraftComputing/ProjectAxeEffect#10]
Received my 2 beta units, went to test physical mounting options and realized that the spacing for the 1U rackmount are spot on, however the holes themselves are slightly too small to use with RackStud mounting hardware..... It looks like the holes are sized to handle M8 screws at largest, RackStuds are M10 in size.....

**Comment by IndigoHaze** [comment]
Additional Data:

Holes are bigger than 7/32 (5.5563mm)

Using a 15/64 (5.9531mm - Increase of less than 0.4mm) drill bit to shave a very small amount off the holes resulted in the RackStuds Duo being a snug but functional fit (easy enough to put on the stud, bit harder to pull off the stud)

The "classic" Maroon/Purple Studs however do not fit through that hole but the will screw into it. Stepping up to 1/4 (6.3500 mm) provides a nice fit for all of the RackStud options....

So with a minor diameter of roughly 5.9531mm it would appear that the Classic Rack Studs are about a M7 screw equiv while the RackStud Duo with a major less than 5.9531mm would be around a M6....

And the sizing info I had before was wrong in that the M10 sizing noted on their website is for the Nut not the Shank of the system... so a 10mm Nut being used for what measures out to be between a M6 and M7 Shank.....

**Comment by CraftComputing** [comment]
Yes, I sized the holes for M8. I was not aware newer RackStuds were a larger diameter. I'll see about adding a case STL for larger screws.

**Comment by IndigoHaze** [comment]
Double check the sizing you used.... if you sized for M8 it should fit (varying for print tolerance, I'm working with data sample size of 2 printed units) both variants... Might be that you sized based off the Minot Diameter instead of the Major Diameter/Clearance Diameter of the M8 screw size?

**Comment by thefathacker** [comment]
Ran into the same issue, ended up just using a deburring tool shave it down JUST enough
